daal4py/sklearn/ensemble/tests/test_decision_forest.py

```diff
@@ -75,15 +77,15 @@
 ]
 
 
-def check_classifier_class_weight_iris(weight):
+def check_accuracy_classifier_class_weight_iris(weight):
     for _ in range(N_TRIES):
         x_train, x_test, y_train, y_test = \
-            train_test_split(IRIS.data, IRIS.target, 
-                             test_size=0.3, random_state=31)
+            train_test_split(IRIS.data, IRIS.target,
+                             test_size=0.33, random_state=31)
 
-        scikit_model = ScikitRandomForestClassifier(class_weight=weight, 
+        scikit_model = ScikitRandomForestClassifier(class_weight=weight,
```

Comment:
Is algorithm parameters absence needed? It's better to specify to avoid problems when their default values are changed
Applicable for other checks too